recipes/calliope/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "calliope" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "78c483e996d3f51f6285a505a82df002ac3079f33894563b2c5ec08dea265025" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<35]
+  entry_points:
+    - {{ name }} = {{ name }}.cli:cli
+
+requirements:
+  build:
+    - basemap
+    - bottleneck
+    - click >=3.3
+    - glpk ==4.59
+    - numexpr >=2.3.1
+    - numpy >=1.10.1
+    - pandas >=0.17.1,<0.18
+    - pyomo >=4.2,<4.3
+    - python >=3.5
+    - pyyaml >=3.11
+    - tables >=3.2.0
+  run:
```

Comment:
There are some packages that are listed in `install_requires` in the `setup.py` that are missing here: `xarray` and `netcdf4`

There are also libraries that are in the `requirements.yml` that aren't included here, like `shapely`, `descartes`, `geos`, `scipy`.

Please check to ensure you have the complete list of requirements.